Config/CommonConfiguration.swift

```diff
@@ -93,11 +93,14 @@ class CommonConfiguration: NSObject, Configurable {
 
         sdkOptions.enableNewClientInformationFeature = RiotSettings.shared.enableClientInformationFeature
         
-        #if DEBUG
         if sdkOptions.isCryptoSDKAvailable {
-            sdkOptions.enableCryptoSDK = RiotSettings.shared.enableCryptoSDK
+            let isEnabled = RiotSettings.shared.enableCryptoSDK
+            MXLog.debug("[CommonConfiguration] Crypto SDK is \(isEnabled ? "enabled" : "disabled")")
+            sdkOptions.enableCryptoSDK = isEnabled
+            sdkOptions.enableStartupProgress = isEnabled
+        } else {
+            MXLog.debug("[CommonConfiguration] Crypto SDK is not available)")
```

Comment:
@NicolasBuquet FYI this configuration is related to the new Rust based Crypto SDK. 
`enableCryptoSDK` should stay false until you decide to switch on this new Crypto SDK. You're using the previous one by default. Let us know if you need more details on this new Crypto SDK

You may have a look on `sdkOptions.isCryptoSDKAvailable`. This flag is true by default now at the matrix-ios-sdk level (https://github.com/matrix-org/matrix-ios-sdk/blob/develop/MatrixSDK/MXSDKOptions.m#L58). You should be able to keep the control with `enableCryptoSDK` flag

Reply:
Thank you. I updated the PR with your advice.